sources/packages/web/src/services/http/dto/Application.dto.ts

```diff
@@ -33,6 +33,13 @@ export interface InProgressApplicationDetailsAPIOutDTO {
 }
 
 export interface SaveApplicationAPIInDTO {
+  programYearId: number;
+  offeringIntensity: OfferingIntensity;
+  data: unknown;
+  associatedFiles: string[];
+}
+
+export interface UpdateApplicationAPIInDTO {
```

Comment:
API DTOs must mirror the backend. Could you make sure a similar one is used in the API?